features/upgrade/node/node-upgrade.feature

```diff
@@ -16,16 +16,18 @@ Feature: Node components upgrade tests
     And I wait up to 30 seconds for the steps to pass:
     """
     When I run the :get client command with:
-      | resource | machineconfigpool |
+      | resource      | machineconfigpool |
+      | resource_name | master            |
     Then the output should match:
-      | master.*False\\s+True\\s+False |
+      | .*False\\s+True\\s+False |
```

Comment:
I even don't know what this check should mean by reading the scenario. Also human readable strings and order of elements may change. Better write a special step to check what is needed here. 

I will not block merging as this is same as before. But consider updating it when possible.